easy/Chalange1.js

```diff
@@ -0,0 +1,12 @@
+//Create a function that takes a number as an argument. Add up all the numbers from 1 to the number you passed to the function.
+
+function add(num)
+{
+let temp = num;
+while(num>0)
+{
+ temp=temp+(num-1);
+ num--
+}
+document.write(temp)
```

Comment:
Instead of `document.write`, Let's start to return output from function and print it where we call this function

Reply:
changes updated 
commit Id fde1f9309d8fe7c99e5b45bddf82086d41eccc52